ccia/Containerfile

```diff
@@ -1,5 +1,10 @@
 ARG BASE_NAME=registry.fedoraproject.org/fedora-minimal
-ARG BASE_TAG=latest
+# FIXME FIXME FIXME! 2023-11-09: revert "38" to "latest"
+# ...38 is because as of this moment, latest is 39, which
+# has python-3.12, which causes something to barf:
+#         aiohttp/_websocket.c:3744:45: error: ‘PyLongObject’ {aka ‘struct _longobject’} has no member named ‘ob_digit’
+# Possible cause: https://github.com/cython/cython/issues/5238
+ARG BASE_TAG=38
```

Comment:
Oof, another one.  Maybe add the time-bomb to `ci/validate.sh` based on a search of all files in the repo?

Stepping-back, my main concern here is the adage "There's nothing more permanent than temporary".   Maybe this is an annoyance that can be considered later and/or in a separate PR.

Reply:
Cannot add `timebomb` here... and am reluctant to add a complicated special-case to validation. It would need to find `FIXME`s, then look for a YMD, then parse, then just yuk. Some day this will blow up again, and whoever fixes it can revert to `latest` (if that's really desirable) or `f39` or do whatever is necessary then. This really can't be futureproofed.

Reply:
Okay, thanks for your considerations and explaining your thinking.